lib/state/simperium/middleware.ts

```diff
@@ -351,6 +351,11 @@ export const initSimperium = (
         return result;
       }
 
+      case 'RESTORE_TAGS': {
+        action.tags.forEach((tagHash) => queueTagUpdate(tagHash));
+        return result;
+      }
```

Comment:
see comments above: this code is good, but we can do everything we need just by listening to the existing note-revision-restore action.

Reply:
Good point! I've modified this in the following [commit](https://github.com/Automattic/simplenote-electron/pull/2817/commits/728e23fbbb743c472de0118744014d4281d97e47), for this part I had to calculate the tags to restore through the previous state, the same as it's done in the `data` reducer.